crml/staking/src/lib.rs

```diff
@@ -1373,6 +1373,11 @@ impl<T: Trait> Module<T> {
 		imbalance
 	}
 
+	#[cfg(any(feature = "std", test))]
```

Comment:
Not sure if this is frowned upon to have a function guarded behind test. But there are some examples from plug here:: https://github.com/plugblockchain/plug-blockchain/blob/develop/frame/system/src/lib.rs#L853